docs/docs/contribute/connectors.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Connectors"
+description: Details the Meltano Connector Quality Standards
+layout: doc
+sidebar_position: 5
+sidebar_class_name: hidden
+---
+
+## Connector Maintainers
+
+- **Official** - Official connectors and tools are built, maintained, supported, and tested by Meltano.
+- **Partner** - Partner connectors and tools are built, maintained, supported, and tested by a partner organization of Meltano, including consultancies and other vendors.
+- **Community** - Community connectors and tools are built and maintained by the wider open source community.
+
+## Connector Quality
+
+- **Gold** - Connectors built on the Meltano SDK with high usage, and broad coverage of data and features.
+- **Silver** - Connectors built on the Meltano SDK with low to medium usage, and connectors not built on the Meltano SDK with high usage.
+- **Bronze** - Connectors not built on the Meltano SDK with low to medium usage.
+- **No Data** - Connectors for which we have no usage data. These may be poorly tested or out of date, and repository maintainers may be unresponsive.
+
+### Connector Quality Matrix
+
+To give users a better understanding of the overall quality of a connector, we have the following matrix and guidance on attributes that affect the quality rating.
+
+|         | SDK-Based | Usage Data    | Maintainer                      | Repo Responsiveness |
+| ------- | --------- | ------------- | ------------------------------- | ------------------- |
+| Gold    | Yes       | > 5 Projects  | Official, Partner, or Community | High                |
+| Silver  | Possibly  | >= 1 Projects | Partner or Community            | Medium              |
+| Bronze  | No        | >= 1 Projects | Community                       | Low                 |
+| No Data | No        | Unknown       | Community                       | Unknown             |
```

Comment:
The only thing that feels more or less vibes-based is "Responsiveness". It's ok as is, but, for example: could an SDK-based, highly used, Community connector, with low responsiveness still be considered Gold-quality?

Reply:
Ok, just wanted to confirm. Fine by me.